core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1441,6 +1441,32 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
   }
 
   /**
+   * Delete a file to be downloaded with this Spark job on every node.
+   * The `path` passed can be either a local file, a file in HDFS (or other Hadoop-supported
+   * filesystems), or an HTTP, HTTPS or FTP URI.  To access the file in Spark jobs,
+   * use `SparkFiles.get(fileName)` to find its download location.
+   *
+   */
+  def deleteFile(path: String): Unit = {
```

Comment:
Hello Reynold: Sorry I am afraid that I misunderstood your previous comments. Does your mean the user should take the path from the LIST FILE command output, then use that path as the DELETE FILE command's path? If that is the case, the delete code will much simple. Thanks for your advice.